model.py

```diff
@@ -0,0 +1,53 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import numpy as np
+import math
+
+
+class heuristic:
+    """
+    Heuristic model that just return the sigmoid of the difference in gold and experience combined,
+    with scaling and bias.
+    """
+    
+    def __init__(self,xp_scale_factor=1.0,total_scale_factor=10.0):
+        self.xp_scale_factor = xp_scale_factor
+        self.total_scale_factor = total_scale_factor
+        self.gold_xp_scale = torch.cat((torch.ones(10),self.xp_scale_factor*torch.ones(10)))
+        
+    def fit(self, inputs):
+        gold_xp_combined = torch.sum(inputs,1)
+        return self.sigmoid(gold_xp_combined/self.total_scale_factor)
+    
+    def sigmoid(self, inputs):
+        return 1/(1+torch.exp(-inputs))
+
+
+class LSTM_baseline(nn.Module):
```

Comment:
To make this module more modular, you should have all models follow a uniform format. I.E., they should all have a `fit()` and `inference()` method implemented. This way, when you want to try a different model, you just swap it out in the code and it will "just work." Otherwise, you'll need a custom notebook for every model, and that just doubled the amount of work you need for testing different models.